optaplanner-examples/src/main/java/org/optaplanner/examples/meetingscheduling/app/MeetingSchedulingApp.java

```diff
@@ -48,7 +49,7 @@ protected MeetingSchedulingPanel createSolutionPanel() {
 
     @Override
     public SolutionFileIO<MeetingSchedule> createSolutionFileIO() {
-        return new XStreamSolutionFileIO<>(MeetingSchedule.class);
+        return new MeetingSchedulingXlsxFileIO();
```

Comment:
This PR adds the .xlsx files, which is good. But it doesn't remove the .xml files, so those linger around, confusing users. Delete those .xml files in the unsolved directory too.

Adjust the .gitignore in the solved directory if you haven't already.